recipes/boost/all/test_package/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 PROJECT(MyHello)
-cmake_minimum_required(VERSION 2.8)
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Why do you need targets? Believe or not, cmake 2.8 is largely used yet, this change can break users.

Reply:
That's why I was hesitating and asking before. Imported targets give better error checking, but demand a newer cmake version. It's a trade off. I can also delete the commits again if you prefer.

Reply:
I think you can decrease to Cmake 3.1 at least. Targets are available on that version. Thanks

Reply:
OK, I was only referring to the cmake documentation. will do that